addons/web/static/tests/helpers/utils.js

```diff
@@ -364,12 +364,12 @@ export async function triggerScroll(
     await nextTick();
 }
 
-export function click(el, selector, skipVisibilityCheck = false) {
+export function click(el, selector, skipVisibilityCheck = false, options = {}) {
```

Comment:
should not be needed either (I am referring to the change I suggested in order to keep the shiftkeymode state in the list renderer, with keydown/up events)